Objects/StrategyCardAutomator.ttslua

```diff
@@ -276,15 +315,92 @@ function onLoad()
 		end
 	end
 
-	self.setColorTint("Green")
-
-	self.UI.setXmlTable(objectUI)
+	--Description on Back of the Object.
+	self.UI.setXmlTable({backManual})
+	--self.tooltip = "TESTTOOLTIP"
```

Comment:
Tooltips on in-game UI (vs global) are hard to read sadly.  Instead of commenting out code maybe leave a comment to that effect? (Or just delete the commented code.)